.github/dependabot.yml

```diff
@@ -16,5 +16,21 @@ updates:
       - "src/csi-wrapper"
       - "src/peerpod-ctrl"
       - "src/webhook"
+    groups:
+      aws-sdk-go-v2:
+        patterns:
+          - "*aws-sdk-go-v2*"
+      k8s:
+        patterns:
+          - "*k8s.io*"
+      x/crypto:
+        patterns:
+          - golang.org/x/crypto
+      x/net:
+        patterns:
+          - golang.org/x/net
+      x/sys:
+        patterns:
+          - golang.org/x/sys
```

Comment:
The question is whether we expect individual groups to break think often. If not then I'd personally prefer one big group update meaning one PR to review and test. On the other hand, bisecting a regression would become harder as one would have to split the commit which brings us back to the original question, do we expect breakages often?

Reply:
I don't have a strong opinion about this but so far I usually saw mass-updates of go.mod files here. So to simulate the same behaviour we could use one group.